packages/elements/src/number-field/__test__/number-field.test.js

```diff
@@ -306,6 +306,101 @@ describe('number-field/NumberField', function () {
       await elementUpdated(el);
       expect(el.value).to.equal('0', 'value should not be less then zero');
     });
+    it('Should fires up-click and value-changed when keydown spinner up', async function () {
```

Comment:
I got confusing between `up-click` and `value-changed`, their are similar behavior but different naming pattern 😵‍💫